build/vsts-ci.yml

```diff
@@ -38,6 +44,66 @@ phases:
       artifactName: PackageAssets
       artifactType: container
 
+################################################################################
+- phase: Linux arm
+################################################################################
+  variables:
+    BuildConfig: Release-netcoreapp3_1
+    OfficialBuildId: $(BUILD.BUILDNUMBER)
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+    DOTNET_MULTILEVEL_LOOKUP: 0
+  queue:
+    name: DotNet-Build
+    demands:
+    - agent.os -equals linux
+    container: UbuntuCrossArmContainer
+  steps:
+  - script: ./restore.sh
+    displayName: restore all projects
+  - script: ./build.sh -configuration $(BuildConfig) /p:CopyPackageAssets=true /p:SkipRIDAgnosticAssets=true -projects $(Build.SourcesDirectory)/src/Redist/Microsoft.ML.DnnImageFeaturizer.ModelRedist/Microsoft.ML.DnnImageFeaturizer.ModelRedist.proj
+    displayName: build redist
```

Comment:
>redist [](http://example.com/codeflow?start=23&length=6)

should you specify here arm32? and in arm64 phase you specify arm64?

Reply:
For the display name? Or which part? The display name in arm64 doesn't have it either.

Reply:
Yes, I was referring at the display name. when the display name gets used? is it in the logs? I was just suggesting adding the specific architecture as more information if someone looking at the logs. If this is not really useful then you can ignore this comment.

Reply:
Its used in the build pipeline. We will can the name of the step without expanding the tab.

I don't think we need it since each build itself is only for one architecture. For example if its the Mac x64 build thats all that we build for in any of the steps.